chains/tendermint/light.go

```diff
@@ -164,10 +164,12 @@ func (pr *Prover) GetLightSignedHeaderAtHeight(height int64) (*tmclient.Header,
 		return nil, err
 	}
 
-	protoVal, err := tmtypes.NewValidatorSet(sh.ValidatorSet.Validators).ToProto()
+	valSet := tmtypes.NewValidatorSet(sh.ValidatorSet.Validators)
+	protoVal, err := valSet.ToProto()
 	if err != nil {
 		return nil, err
 	}
+	protoVal.TotalVotingPower = valSet.TotalVotingPower()
```

Comment:
Does this change fix a bug in the original version?

Reply:
It's required to support tendermint-rs's verifier. The verifier checks if `TotalVotingPower` matches computed one from each power.
https://github.com/informalsystems/tendermint-rs/blob/d102af4430900f27f8113bd8a048bfb22cba3a7d/tendermint/src/validator.rs#L45